Rebuild-Black-Business/RBB-Website
# Fix/allies ssr grid

# Describe your PR

Fixes #129

## Pages/Interfaces that will change

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1. go directly to build of `/allies`
2. check if grid looks same as going there via the menu

### Additional notes
